### DISCUSSION TITLE
How to find where a URL is used in a wiki

### DISCUSSION THREAD

**Discussion by lbass44** [requarks/wiki#7186] [Help / Questions]
We were able to pull a list of all the links in our wikis and I am going through them to find any that are broken. The issue is, I don't know how to find which wiki the link was used in. The search function doesn't allow you to search by URL. Any ideas?

**Comment by mittkkoo** [comment]
I think Selenium will be perfect for such kind of stuff. You can write a script to browse the wiki, check how links respond etc.
